content/website/blog/index.en.md

```diff
@@ -0,0 +1,170 @@
+Any images, data files or other things you need to include in your post, add them directly in the folder.
+
+As you write you post, remember to `knit` your post so have a look at how it looks like!
+The blogdown site will use the markdown file created from your knitted `Rmd` on the site, not your `Rmd` itself.
+You can preview the entire site with your post with `blogdown::serve_site()`.
+
+### After you write your post
+
+Once your post looks as you want it to on your local machine, it's time to push the post
+up to the main repository for review.
+
+### via git in the terminal
+
+```sh
+# Add the post
+git add content/post/yyyy-mm-dd-your-post-title
+
+# Add a commit message to the post
+git commit -m "my commit message"
+
+# Push changes online (replace my_branch with your branch name)
+git push --set-upstream origin my_branch
+```
+
+### via {usethis}
+
+In the RStudio git pane, check your post folder and commit with a message.
+
+Then push with
+
+```r
+usethis::pr_push()
+```
+
+## PR your changes
+
+Once you have pushed your changes online, make a [Pull request (PR)](https://github.com/rladies/blog/pulls) to the main branch.
+If you are not a member of the R-Ladies Global team, you will be working on a "fork" of the repository.
+
+When the PR is open ([see here](https://github.com/rladies/blog/pulls)), a person from the blog or website team will be assigned as your reviewer and will take you through the remaining process.
+
+When working on a fork, the build of the website will run, but no preview will be created.
+
+### Website/Blog administration take-over
+
+After the PR is received, someone in the @rladies/blog or @rladies/website team will have a look at your post.
+They will request changes through GitHub code review, where you will need to adapt or accept the changes they propose to your document.
+
+Once the team is satisfied with the post, they will [create a new branch from the main repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository#creating-a-branch) and call it something like "fork\_[newpostname]", and then [switch your PR's base branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request) to the newly made fork rather than `main`.
+They will then merge your PR into this branch, and wait for a preview to be generated.
+
+Ideally, at this point, only other changes that might need to be done are editorial and up to the @rladies/blog or @rladies/website team to fix.
+They will ask for you assistance directly if anything else is needed.
+
+## Clarification
+
+If anything is unclear in these guidelines, please submit [an issue](https://github.com/rladies/blog/issues), so that we can assist you and improve our documentation.
```

Comment:
```suggestion
If anything is unclear in these guidelines, please submit [an issue](https://github.com/rladies/rladiesguide/issues), so that we can assist you and improve our documentation.
```